### ISSUE TITLE
Make CheckBox Accessible

### ISSUE THREAD

**Issue by sydneywalcoff** [sydneywalcoff/app-tracker#175]
### Overview
Our checkbox component is currently not accessible to users navigating by keyboard or screen readers.

### Action Items
- [ ] make checkbox component accessible
   - [ ] changed with Space key

### Resources/Instructions

Related Issue: 
- #163

**Comment by sydneywalcoff** [comment]
made redundant by #237
